modules/posenc/posenc.timing.ini

```diff
@@ -0,0 +1,7 @@
+[.]
+description: Timing diagrams for the posenc block
+scope: posenc.block.ini
+
+[test]
```

Comment:
Can you write some tests please, the idea of this block is to take a position and output step and direction output or quadrature output